style.css

```diff
@@ -33,7 +34,8 @@
   font-family: Courier;
   font-weight: 1000;
   font-size: 1rem;
-  color: var(--black)
+  color: var(--black);
+  text-shadow:0px 0px 2px var(--white);
```

Comment:
```suggestion

```
Maybe we could do without text shadow? White shadow on white texts gives a kinda glowy/blurry effect for the chum/memo lists which idnk if is desirable.

![image](https://github.com/Dpeta/PesterchumOnlineButSilly/assets/69427753/09dd593f-5c67-487e-b298-6dd85f88a9ee)
![image](https://github.com/Dpeta/PesterchumOnlineButSilly/assets/69427753/79e99a38-b9b0-4f7c-a3f5-e40edc337216)

![image](https://github.com/Dpeta/PesterchumOnlineButSilly/assets/69427753/57cd6b65-8eb9-41f2-8e0c-9e3414d45298)
![image](https://github.com/Dpeta/PesterchumOnlineButSilly/assets/69427753/8e9e4f94-40cf-4365-86b8-a023ad0186ef)

don't mind memo/chums tab buttons though with the darker text & background
![image](https://github.com/Dpeta/PesterchumOnlineButSilly/assets/69427753/77d7978e-fe8c-486f-91d2-572ba761efe1)
![image](https://github.com/Dpeta/PesterchumOnlineButSilly/assets/69427753/d15ac4c8-cc79-4b93-bf52-787471e3e7fc)